src/background.ts

```diff
@@ -148,6 +146,19 @@ const store = new Store<ElectronStoreType>({
           defaultStyleId: defaultStyle.defaultStyleId,
         }))
       );
+
+      const outputSamplingRate: number =
+        // @ts-expect-error 削除されたパラメータ。
+        store.get("savingSetting").outputSamplingRate;
+      store.set(`engineSetting.${engineId}`, {
```

Comment:
あ、engineSettingsかもです！
```suggestion
      store.set(`engineSettings.${engineId}`, {
```